integration/helpers/utils.ts

```diff
@@ -13,7 +13,6 @@ export function setup(): Application {
     path: AppPaths[process.platform],
     startTimeout: 30000,
     waitTimeout: 60000,
-    chromeDriverArgs: ['remote-debugging-port=9222'],
```

Comment:
Why this was removed?

Reply:
I removed this when fixed integration tests since it did not make any difference